packages/geo/src/lib/feature/shared/feature.utils.ts

```diff
@@ -50,6 +50,7 @@ export function featureToOl(
   const title = getEntityTitle(feature);
   if (title !== undefined) {
     olFeature.set('_title', title, true);
+    olFeature.title = title;
```

Comment:
"title" isn't an attribute of ol.Feature. Why is this needed?

Reply:
To assign a significant title at each layer added to the map (logic in the tool component).

Reply:
That's what properties are for. Can't you do olFeature.get('_title')

Reply:
Seems logic, will try !

Reply:
done